yt_dlp/extractor/cspan.py

```diff
@@ -242,3 +243,49 @@ def get_text_attr(d, attr):
                 'title': title,
                 'id': 'c' + video_id if video_type == 'clip' else video_id,
             }
+
+
+class CSpanCongressIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?c-span\.org/congress/'
+    IE_DESC = 'C-SPAN'
+    _TESTS = [{
+        'url': 'https://www.c-span.org/congress/?chamber=house&date=2017-12-13&t=1513208380',
+        'info_dict': {
+            'id': 'Congressional Chronicle - Members of Congress, Hearings and More | C-SPAN.org',
```

Comment:
i don't like this id tbh.
I personally would have parsed the `chamber` `date` and `t` from the url and maybe hashed them if not just joined them  using `_` and used that as the id.

Reply:
+1, the webpage title seems static across all videos so this would completely break archiving